ts/features/pn/components/MessageDetails.tsx

```diff
@@ -148,12 +149,12 @@ export const MessageDetails = ({
           presentPaymentsBottomSheetRef={presentPaymentsBottomSheetRef}
         />
 
-        {RA.isNonEmpty(f24List) && (
+        {!isCancelled && RA.isNonEmpty(f24List) ? (
```

Comment:
Why not just
```
{!isCancelled && RA.isNonEmpty(f24List) && (<> ... </>)}
```
without the ternary?

Reply:
I think the ternary is more readable, but there is no difference. What do you think?